example-config.yml

```diff
@@ -268,7 +268,11 @@ modes:
       iconName: wheelchair
       type: CHECKBOX # Possible options: CHECKBOX, SUBMODE, SLIDER, DROPDOWN
   transitModes:
-    - mode: BUS
+      # Mode can be a string or an array of strings.
+    - mode: ["TROLLEYBUS", "BUS"]
+      # When mode is an array an overrideMode must be provided.
+      # This specifies the displayed mode used for icon and URL param.
+      overrideMode: BUS
```

Comment:
This key title is confusing. But I can't think of a better one

Reply:
Was trying not to make it a breaking change.

Reply:
we could've done a thing where we just use the first item in the array as the override but this is fine too!

Reply:
that's not a bad idea! we'll see what second reviewer thinks

Reply:
The name can be confusing at first. Maybe change `overrideMode` to `summaryMode` or `displayedMode` but I don't feel strongly about it. It looks like branded names are already supported, so that should be fine.

Reply:
I wanted to keep it so we didn't change the config at all. I agree that the name choice wasn't great but I don't think it's worth updating now.